substrate/client/block-builder/src/lib.rs

```diff
@@ -255,6 +255,15 @@ where
 		})
 	}
 
+	/// Called after inherents but before extrinsics have been applied.
```

Comment:
```suggestion
	/// Called after inherents but before transactions have been applied.
```

inherents are a type of extrinsic. extrinsic simply means "data from the outside".